pkg/template/generator/expressionvalue.go

```diff
@@ -94,6 +95,8 @@ func replaceWithGenerated(s *string, expression string, ranges [][]byte, length
 			alphabet += Numerals
 		case `\a`:
 			alphabet += Alphabet + Numerals
+		case `\S`:
```

Comment:
`\S` normally means non-whitespace in regex... why are we using that to mean "Non-Alphabet" here? that's unexpected